distributed/cli/dask_ssh.py

```diff
@@ -29,12 +29,13 @@
               help="Port to use for SSH connections.")
 @click.option('--ssh-private-key', default=None, type=str,
               help="Private key file to use for SSH connections.")
-
+@click.option('--bindall', is_flag=True,
+              help="Let the worker bind to all available interfaces.")
```

Comment:
Ah, please note that the use of `--host` is not a bind address, but rather the name by which the worker refers to itself.  

When a worker registers with the scheduler it provides an address that other workers should use to contact it in the future.  That is the meaning of the value provided by `--host`.